### DISCUSSION TITLE
Is there a way to turn off automatic unloading detection, so that I can set a unload point  manually?

### DISCUSSION THREAD

**Discussion by rento112** [Courseplay/courseplay#6889] [Q&A] [answered]
Is there a way to turn off automatic unloading detection, so that I can set a unload point  manually?

**Comment by Tensuko** [comment, marked as answer]
No.
But what do you try to do ?

**Reply by rento112** [reply]
I'm using silos that have the overhead fill on top of the unload area. So it unloads as soon as it finishes loading. :/

**Reply by Tensuko** [reply]
Oh okay I get it.
I think what should help you there is to move the start point more away from the unload area.
If I remember correct, we check distance or waypoints before we try to unload again.
If you have a Link to that Silo, pls give it to me and I have a look.
